src/northbridge/amd/pi/agesawrapper.c

```diff
@@ -302,7 +302,7 @@ AGESA_STATUS agesawrapper_amdinitlate(void)
 	Status = AmdInitLate(AmdLateParams);
 	if (Status != AGESA_SUCCESS) {
 		agesawrapper_amdreadeventlog(AmdLateParams->StdHeader.HeapStatus);
-		ASSERT(Status == AGESA_SUCCESS);
+		printk(BIOS_WARNING, "AmdInitLate returned error!\n");
```

Comment:
@mek-x this looks risky. It may happen that some critical issue happen during late init. What then ? Maybe we should query mailing list about this ASSERT ?